.github/workflows/build_conda_linux.yml

```diff
@@ -0,0 +1,48 @@
+name: Build Linux Conda
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
```

Comment:
I have seen a nightly branch created and updated daily. Wondering if there is a way for us to manual trigger nightly release and all those build workflows?

Reply:
We have a separate Chronos job to trigger a nightly build https://fburl.com/chronos/udouc4ht So we can trigger it also manually.

Reply:
Ohhh, thank you!